Makefile

```diff
@@ -81,7 +81,7 @@ deploy:
 
 .PHONY: deploy-clean
 deploy-clean:
-	go run ./cmd/kubectl-kudo  init --crd-only --dry-run --output yaml | kubectl delete -f -
+	go run ./cmd/kubectl-kudo  init --dry-run --output yaml | kubectl delete -f -
```

Comment:
this is fine... I assume there is awareness that if there is a runtime object removal that this doesn't work.  In the past the removal of planexecution caused some issues as the latest version didn't delete the crd...  but 👍

Reply:
I think the convention should be `XXX-clean` removes whatever XXX created. I doubt that many people actually use this to be honest apart from the core team, I would not be afraid of changing this :)